ledger-api/grpc-definitions/com/daml/ledger/api/v1/event.proto

```diff
@@ -55,9 +57,22 @@ message CreatedEvent {
   Value contract_key = 7;
 
   // The arguments that have been used to create the contract.
-  // Required
+  // Set if there was a match by template_id in the given transaction filter.
+  // Optional
   Record create_arguments = 4;
 
+  // Opaque representation of contract payload intented for forwarding
+  // to an API server as a contract disclosed as part of a command
+  // submission.
+  // Optional
+  google.protobuf.Any create_arguments_blob = 10;
+
+  // Interface views specified in the transaction filter. Only contains entries
+  // for the intersection of interfaces implemented by the template
+  // and the interfaces specified in the transaction filter.
+  // Optional
+  repeated InterfaceView interface_views = 11;
```

Comment:
nit: maybe consider making this PR a PR against the other branch so that the diff only includes explicit disclosure. Obviously becomes redundant once the first one is merged so no need to spend a lot of time on it.

Reply:
thanks, that was the original intention, fixed